eyecite/models.py

```diff
@@ -69,7 +69,9 @@ class CitationBase:
     index: int  # index of _token in the token list
     # span() overrides
     span_start: Optional[int] = None
+    full_span_start: Optional[int] = None
     span_end: Optional[int] = None
+    full_span_end: Optional[int] = None
```

Comment:
Just a nitpick, but I think I'd do it this way:

```suggestion
    span_start: Optional[int] = None
    span_end: Optional[int] = None
    full_span_start: Optional[int] = None
    full_span_end: Optional[int] = None
```

Reply:
I couldn't find where span_start or span_end are assigned, no. Weird. Could be vestigial. I didn't see span() being tested either, but c.span() is used in the readme, so it seems like these are getting set somehow!

Reply:
I was surprised as well, They are probably never set, I think it only works because the `span()` function uses the `start` and `end` attributes of the stored citation's token when `span_start` and `span_end` are not set ([here](https://github.com/freelawproject/eyecite/blob/main/eyecite/models.py#L143-L150))

Reply:
I think you're right. Want to add a fix for this either to remove the attributes or set them?

Reply:
Maybe better to set them at initialization to keep backward compatibility

Reply:
Yeah, that's probably better, you're right. Thank you!